app/controllers/admin/dashboard_controller.rb

```diff
@@ -11,4 +12,12 @@ def prep_calendar
     @division = current_division
     @calendar_events_url = "/admin/calendar_events?person_id=#{@person.id}"
   end
+
+  def prep_logs
+    @context = "dashboard"
+    @logs = ProjectLog.in_division(selected_division).
+        where(agent_id: @person.id).
+        order('date IS NULL, date DESC, created_at DESC').
+        page(1).per(10)
```

Comment:
Just one indent level please. Otherwise, pretty! Though once by_date moves into model, you might be able to one-line this...